src/backend/base/langflow/base/models/model_input_constants.py

```diff
@@ -28,7 +28,7 @@ def process_inputs(component_data):
     if isinstance(component_data, SecretStrInput):
         component_data.value = ""
         component_data.load_from_db = False
-    elif component_data.name == "temperature":
+    elif component_data.name in ("temperature", "tool_model_enabled", "base_url"):
```

Comment:
nit: use set to "performance" hahaha 
```suggestion
    elif component_data.name in {"temperature", "tool_model_enabled", "base_url"}:
```